hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeConfiguration.java

```diff
@@ -188,6 +188,24 @@ public class DatanodeConfiguration {
   private long recoveringContainerScrubInterval =
       Duration.ofMinutes(10).toMillis();
 
+  /**
+   * Timeout for the thread used to process the delete block command
+   * to wait for the container lock.
+   * It takes about 200ms to open a RocksDB with HDD media.
+   * Set the default value to 100ms, so that after one times retry
+   * after waiting timeout, the hold time spent waiting for the lock
+   * is not greater than the time spent operating RocksDB
+   */
+  @Config(key = "block.delete.command.handle.lock.timeout",
```

Comment:
Suggest change the property name  from block.delete.command.handle.lock.timeout -> 
block.delete.max.lock.wait.timeout

Reply:
@xichen01 , is the slow RocksDB open a major cause behind this patch?  I have one question,  it takes 200ms to open a rocksdb, you set this timeout to 100ms, and retry once in the block deletion logic, why not set the default value to 200ms, with a simpler handler logic?

Reply:
Will this property be a reconfigurable property?

Reply:
This comment may have caused some misunderstanding, I'll modify it. 

The retry does not happen immediately, it waits until all `DeleteBlockTransactions` have been processed, so the retry may happen after a few seconds or tens of seconds.

Slow RocksDB open (Schema V2 only), rebalancer, `BlockDeletingService`, etc, may cause here cannot get the lock, retry to increase the chance of success.

Reply:
@xichen01 , thanks for the further info.